cranelift-codegen/meta/src/cdsl/instructions.rs

```diff
@@ -177,8 +177,17 @@ impl Instruction {
         bind(self.clone(), Some(lane_type.into()), Vec::new())
     }
 
-    pub fn bind_vector(&self, lane_type: impl Into<LaneType>, num_lanes: u64) -> BoundInstruction {
-        bind_vector(self.clone(), lane_type.into(), num_lanes, Vec::new())
+    pub fn bind_vector_from_lane(
+        &self,
+        lane_type: impl Into<LaneType>,
+        vector_size_in_bits: u64,
```

Comment:
@sunfishcode, this is how I parameterized binding of vectors like I was talking about in https://github.com/CraneStation/cranelift/pull/868#discussion_r307072094.